### DISCUSSION TITLE
param 'http_session' for AsyncClient in document

### DISCUSSION THREAD

**Discussion by cty512** [miguelgrinberg/python-socketio#933] [Q&A] [answered]
Hello Miguel / Maintainer Team,

This is a gentle reminder,

I noticed that in current document for class AsyncClient(client.Client)

"
    **:param http_session:** an initialized ``requests.Session`` object to be used
                         when sending requests to the server. Use it if you
                         need to add special client options such as proxy
                         servers, SSL certificates, etc.
"

Is it possible to have document updated to mention we can pass **aiohttp.ClientSession** here ?

-
related file:
https://github.com/miguelgrinberg/python-socketio/blob/98f3cb4664ff10c0bb17826b11564644bed99fd6/src/socketio/asyncio_client.py

**Comment by miguelgrinberg** [comment, marked as answer]
Thanks, this is now fixed.
